### DISCUSSION TITLE
Scaffold Clean Architecture

### DISCUSSION THREAD

**Discussion by jakspok** [bancolombia/scaffold-clean-architecture#217] [General]
Hello to everyone in this space Question channel, contributions and support for the Scaffold Clean Architecture plugin

**Comment by edilbertocalderontorres** [comment]
Hello. I have a compile problem because "compileJava.dependsOn validateStructure" is declared in the main.gradle file. When I try to run my project, when compiling it I get the error: "The model module is not valid", but no further details are explained about it.

-How can I to know  what I'm doing wrong?
-What type of validation or validations criteria do the plugin on the model module?

Español:
Hola. Tengo un problema de compilación porque "compileJava.dependsOn validateStructure" está declarado en el archivo main.gradle. Cuando trato de ejecutar mi proyecto, al compilarlo me sale el error: "El módulo del modelo no es válido", pero no se explican más detalles al respecto.
¿Cómo puedo saber  qué estoy haciendo mal?

¿Qué tipo de validación o criterios de validación hace el complemento en el módulo del modelo?


@santitigaga 
@dericop 
@dbuos

**Reply by sanleriver** [reply]
Hola @edytkld, no estoy seguro como hace la validación pero se me ha presentado esta validación y supongo que debes estar tratando de utilizar una dependencia en el dominio lo cual no es posible, intenta quitar la dependencia de main.gradle y agregarla en el build.gradle del subproyecto (pero solo de driven-adapters o de entry-point) creo que esto solucionaria tu problema con el validateStructure
